source/SemVerAnalyzer/config.json

```diff
@@ -3,6 +3,6 @@
 		"ruleOverrides": { }
 	},
 	"nuget": {
-		"repositoryUrl": "https://www.nuget.org/api/v2/package/"
+		"repositoryUrl": "https://api.nuget.org/"
```

Comment:
Is this change going to require that people who use a private feed must also change their URI?  If so, this is a breaking change.

Reply:
I've unlisted 1.5.0 from Nuget.org.  A new PR should be created that just bumps to 2.0.0.

Reply:
👍 Right you are. Ok, I'll make a v2.0.0